zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -876,6 +880,11 @@ components:
           type: object
           additionalProperties:
             type: string
+        priority:
```

Comment:
❓ I see the default value is set here as 50, just to confirm, this is the case when there is no priority, or its null. this will assume 50?

Reply:
In the response, theoretically, you should never have null or unset priority since this is handled elsewhere. Mostly added this for documentation regarding the priority ranges.